tier3/checklist.md

```diff
@@ -0,0 +1,441 @@
+# DSAC OSPO Outbound Review Checklist
+## Tier 3: Public Repository
+### Instructions
+This is a review process to approve CMS-developed software to be released open source.
+If you would like your repository to be released, please complete the following steps.
+
+[Instructions](#instructions)
+
+[State the Benefit(s) of Open Sourcing the Project](#state-the-benefits-of-open-sourcing-the-project)
+
+[State the Risk(s) of Open Sourcing the Project, if any](#state-the-risks-of-open-sourcing-the-project-if-any)
+
+[Questions](#questions)
+
+[Code Review](#code-review)
+
+[Code Analysis](#code-analysis)
+
+[Toolkit](#toolkit)
+
+[Review Licensing](#review-licensing)
+
+[Review Commit History](#review-commit-history)
+
+[Review Repository Hygiene](#review-repository-hygiene)
+
+[Running repolinter on your repository locally](#running-repolinter-on-your-repository-locally)
```

Comment:
We will not include subsections of "Review Repository Hygiene" in the table of contents so you can delete line 27, 29, and 31